### ISSUE TITLE
Update Power Bi App

### ISSUE THREAD

**Issue by proofin1** [maikvandergaag/msft-extensions#563]
**For which extension would you like to add a feature request?**
Power BI Actions in AzureDEvOps
**Question**
Is the Power BI app automatically updated during the report upload?

**Comment by leinoaa** [comment]
No, currently there is no API call even for updating an app, except if you are using PBI deployment pipelines.
